pages/submarine/guide_lmde.mdx

```diff
@@ -2,9 +2,9 @@
 
 ## Partitioning
 
-Ensure that you disk uses a GPT partition table.
+Ensure that your disk uses a GPT partition table.
 
-1. Create a 16mb partition, this is where Submarine will be.
+1. Create a 16mb EXT4 partition, this is where Submarine will be.
```

Comment:
doesn't matter what it is, submarine erases it anyway

Reply:
Everywhere else says ext4, it should be unified. Everywhere else can be changed to not saying ext4.